lib/GWPARAM.ml

```diff
@@ -4,6 +4,11 @@
     for simple functions if it does not come with a performance cost.
 *)
 
+let nb_errors = ref 0
+let errors_undef = ref []
+let errors_other = ref []
+let set_vars = ref []
```

Comment:
are those really helpful and needed?

unless I'm mistaken, because of the fork mechanism of geneweb this will be specific to only one connection

maybe you can compute nb_errors from the other ref when needed to be sure you have the right count and not have to worry about it everywhere.

maybe we can update a list of errors for each variant of syslog_level in `syslog` so every error is saved in those.

Reply:
we have found this count and report very usefull to chase bugs in the templates. The principle being agreed upon, one could certainly improve.